twtr/grails-app/controllers/twtr/AccountController.groovy

```diff
@@ -11,24 +11,28 @@ import static org.springframework.http.HttpStatus.NO_CONTENT
 class AccountController extends RestfulController<Account> {
     static responseFormats = ['json', 'xml']
 
-    def AccountController (){
-        super (Account)
+    def AccountController() {
+        super(Account)
     }
 
     def handle() {
-        respond Account.findByHandle(params.id)
-    }
 
+        Account account = Account.findByHandle(params.id)
+        if (!account) {
+            response.sendError(404)
+            return
+        }
+        respond account
```

Comment:
this one should be account information because it's returning an account based on the handle passed in.